arm-network/2017-06-01/swagger/virtualNetwork.json

```diff
@@ -780,6 +780,13 @@
           "$ref": "./routeTable.json#/definitions/RouteTable",
           "description": "The reference of the RouteTable resource."
         },
+        "privateAccessServices": {
```

Comment:
Yes, it was. I've added spec extractions to clarify the changes, please find them in the top PR's comment.